### PR TITLE
Japanese Nanites, Black Market Item

### PR DESCRIPTION
**Pull request by ghost**
Let just uh... Shifty decollabed, yes yes...
It's time.

### CODE REVIEW

**Comment by Kharkov72** [comment]
don't

**Comment by MadmanMartian** [comment]
This PR isn't even atomic!

**Comment by ghost** [comment]
I will come again another time, and with a vengeance

File: code/datums/black_market_item.dm [hunk 1]
```diff
@@ -256,7 +256,18 @@ for anyone but the person committing mass murder.
 	rocket_launcher.emagged = TRUE
 
 /datum/black_market_item/tech/portalgun
```
Comment (brndd) [inline code/datums/black_market_item.dm:258]: 🤔

**Comment by SonixApache** [comment]
please die

**Comment by ghost** [comment]
I still don't understand the purpose of this when there's already a virus symptom and a chemical that do similar things in a far less cringeworthy manner.

**Comment by ShiftyRail** [comment]
I'm a still a collab

**Comment by bathosbathtime** [comment]
Still unatomic.

**Pull request closed without merging**